src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2853,6 +2879,9 @@ private class ConsoleHostStartupException : Exception
 
 #if !UNIX
         private GCHandle breakHandlerGcHandle;
+
+        private ConsoleControl.ConsoleModes _savedConsoleMode;
```

Comment:
For purely defensive purposes, I suppose, but it would be a logically useless assignment.